edx-platform/pearson-aws-theme/lms/templates/student_account/login_and_register.html

```diff
@@ -35,6 +35,14 @@
             <%static:include path="student_account/${template_name}.underscore" />
         </script>
     % endfor
+
+    <style>
```

Comment:
Why do you need to add this here? @kuipumu

Reply:
I applied this style override so we can change .content-wrapper class only on register and login form, so it uses the whole width of the screen, it also removes padding and margin to allow the background image to cover the whole background

This is how it looks if you open it on a screen larger than 1920px:

![image](https://user-images.githubusercontent.com/8495728/195202680-e6d70c45-d69a-4a25-af0c-2ea6cee69ca3.png)